src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttributeGroupController.php

```diff
@@ -308,6 +308,16 @@ private function getErrorMessages()
                     'Admin.Notifications.Error',
                     [sprintf('"%s"', $this->trans('Name', 'Admin.Global'))]
                 ),
+                AttributeGroupConstraintException::EMPTY_PUBLIC_NAME => $this->trans(
```

Comment:
Adding a constraint here and in the CQRS command is a good idea, but you should also add a constraint on the form type This way the error is caught early and displayed in a form inline error It doesn't mean the check in CQRS is useless, it's just another layer of validation, which you should check in the behat tests by the way